### PR TITLE
07_00_ARC07_3F

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Arcueid/Day 7/07_00_ARC07_3F.txt [hunk 1]
```diff
@@ -982,17 +1106,21 @@ After all, I've already admitted it from my own mouth just now.
...
+// Feels like there has to be a better way to encode the "this", "this", wtf is "this" vibe
+Explain yourself!
+ You've been going on about this and that for a while, but what are you trying to say?!"
 }
 [sha:c53d6dd33a657d58a4b9bff5a8e18d90ef483562]{
 -- Page 414, Offset 17605. Mods: @k, @e.
 -- 「だから！
-"I'm saying...
+// そう言いたいんだよ、俺は yoinked from next line
+"What I want to say is this:
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3F.txt:1117]: This is too soft for the exasperated va here imo
>Here's what I want to say:

File: script/Arcueid/Day 7/07_00_ARC07_3F.txt [hunk 2]
```diff
@@ -1067,28 +1197,35 @@ The moment I uttered those words, I lost every right I had to lie to myself.
...
 -- 「早く。わたし、今の台詞もう一度聞きたい」
-"Come on. I want to hear that line just now one more time."
+"Come on. I want to hear that line just one more time."
 }
 [sha:1cf05ed018a7b49dce0de4a37b01322bd4e4b3cb]{
 -- Page 420, Offset 17622.
 -- 囁くように要求してくるアルクェイド。
-Arcueid makes a request to me like a whisper.
+// 要求 has a bit of strength to it, but 'demand' is too much
+Arcueid continues to entreat me, her voice almost a whisper.
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3F.txt:1206]: maybe supplicate? entreat is a little archaic imo

File: script/Arcueid/Day 7/07_00_ARC07_3F.txt [hunk 3]
```diff
@@ -1098,12 +1235,13 @@ I can't rely on someone as weak as you to do it alone, so that's why I offered t
...
 -- Page 422, Offset 17627.
 -- アルクェイドは、ぱあ、と目を輝かした。
-Arcueid's eyes light up, sparkling.
+Arcueid's eyes light up.
 }
 [sha:fe96e84373da3090004dcab5761e499b34a35619]{
 -- Page 422, Offset 17628.
 -- そのまま、嬉しそうにこっちの手を握って、ぶんぶんと上下に振り回す。
-She grabs my hands and shakes them up and down with glee.
+// Doesn't say 両手 here so... assuming just the one hand for all these?
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3F.txt:1243]: that's not necessarily a great assumption given how damn flaky people are with this, but I think in this case you're right.

**Review by Avuxo — APPROVED** [review]
great work on this one. Now on to 7G, sisyphus :aokosmug:

File: script/Arcueid/Day 7/07_00_ARC07_3F.txt [hunk 4]
```diff
@@ -252,37 +274,46 @@ The victims that you people are fussing over are only a small fraction of those
...
+// Duped 百人 over simply 人々
+Over a hundred victims.
+ Over a hundred people attacked, drained of their blood and finally killed
+ by these things that were once their fellow humans.
 }
 [sha:1192a0fd2e80edf3dcd7210c57578d1cfd7ccc9d]{
 -- Page 358, Offset 17464.
 -- そして血を吸われた犠牲者は新たな怪物となって、先ほどの線の塊のように、動く<屍|しかばね>として街を徘徊している―――
-And those victims that had their blood sucked turned into new monsters. Moving corpses roaming the streets like the cluster of lines I had seen earlier―――
+Those victims that have their blood sucked then become new monsters in turn,
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Arcueid/Day 7/07_00_ARC07_3F.txt:314]: Sentence break ok, but not sure I agree with the change to the second sentence - "Ambulant corpses themselves go on the prowl in the city" doesn't really make sense to me

File: script/Arcueid/Day 7/07_00_ARC07_3F.txt [hunk 5]
```diff
@@ -342,77 +381,97 @@ I picture Akiha's body, discarded like garbage after having her blood sucked out
...
 -- Page 363, Offset 17476.
 -- 脳が削られるような目眩を、排気する事でやり過ごす。
-The nausea that seems to be eating away at my brain overcomes my feeling of exhaustion.
+The feeling of vertigo, as though my brain were being slowly whittled away, gives way to exhaustion.
 }
 [sha:a774c8b9e5dd44ac6996947e7c1d06d26a553501]{
 -- Page 363, Offset 17477.
 -- 本来なら<憤|いきどお>るべき心は妙に冷めていた。
-My heart, which should have been rightfully furious, is strangely calm.
+My heart, which ought have been burning with indignation, is strangely calm.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Arcueid/Day 7/07_00_ARC07_3F.txt:389]: This isn't british this is just me accidentallying a word lol

File: script/Arcueid/Day 7/07_00_ARC07_3F.txt [hunk 6]
```diff
@@ -342,77 +381,97 @@ I picture Akiha's body, discarded like garbage after having her blood sucked out
...
 -- 今は震えるより、考えるべき時だと自分に言い聞かせる。
-I tell myself that now is the time to think, not to quiver.
+I tell myself that now is the time to think, not to tremble.
 }
 [sha:aabaa4695d915eebae40ffd38cc94b9656df0ef2]{
 -- Page 364, Offset 17479.
 -- そんな出来事が明日にも起こりうる状況を認識する。
-I recognize that these kinds of events could happen as soon as tomorrow.
+I'm cognizant of the fact that under these circumstances,
+ such an event could still come to pass.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Arcueid/Day 7/07_00_ARC07_3F.txt:400]: Come to pass is a phrase, but it's definitely less immediate. Will change

File: script/Arcueid/Day 7/07_00_ARC07_3F.txt (hunk 2 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 7/07_00_ARC07_3F.txt:1206]: Hmm, supplicate reads a little _too_ grovel-y to me. Implore?

Comment (rschlaikjer) [reply script/Arcueid/Day 7/07_00_ARC07_3F.txt:1207]: Wheedle or cajole perhaps? Or rearrange to something like
> Arcueid's coaxing voice is little more than a whisper.

Comment (Avuxo) [reply script/Arcueid/Day 7/07_00_ARC07_3F.txt:1207]: That last one works well

File: script/Arcueid/Day 7/07_00_ARC07_3F.txt (hunk 4 above)
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Arcueid/Day 7/07_00_ARC07_3F.txt]: yeah, not so great. I was trying to make it more clear but i guess its fine without

File: script/Arcueid/Day 7/07_00_ARC07_3F.txt [hunk 7]
```diff
@@ -66,32 +69,32 @@ Well then! Without further ado, allow me to explain what I was up to, as request
...
 -- Page 342, Offset 17424.
 -- 「じゃあ確認からさせてくれ。
-"In that case, let me confirm this.
+"Alright, let me confirm something first.
 }
 [sha:e20632cc3064b1db4ef30ed0f87479227fd88555]{
 -- Page 342, Offset 17425.
 -- さっきのヤツは吸血鬼の手足になってる死者だな？
-That guy from earlier was a corpse that was controlled by a vampire, wasn't he?
+That guy from earlier was one of those Dead that act as a vampire's worker drones, wasn't he?
```
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Arcueid/Day 7/07_00_ARC07_3F.txt]: I can't come up with anything good. Maybe leave a comment for future proofreaders.

**Pull request merged**